targets/PythonSdk/templates/setup.py.ejs

```diff
@@ -0,0 +1,22 @@
+import setuptools
+with open("README.md", "r") as fh:
+        long_description = fh.read()
+        
+setuptools.setup(
+    name="playfab",
+    version="<%- sdkVersion %>",
+    author="PlayFab Dev Tools team",
+    author_email="playfabdt@microsoft.com",
```

Comment:
This is honestly worse than your own work email.
It lets the public spam us ALL directly (or more hopefully, it's a private alias that outsiders won't be able to mail at all, so it'll just bounce).

If an email is absolutely required, let's use the public zendesk email:
devrel@playfab.com
But, if the email field is not required, then it should just be deleted.